include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -138,6 +138,12 @@ typedef void(^indexstoredb_delegate_event_receiver_t)(_Nonnull indexstoredb_dele
 /// Returns true to continue.
 typedef bool(^indexstoredb_unit_info_receiver)(_Nonnull indexstoredb_unit_info_t);
 
+/// Returns true to continue.
+typedef bool(^indexstoredb_files_included_receiver)(const char *_Nonnull targetPath, size_t line);
+
+/// Returns true to continue.
+typedef bool(^indexstoredb_files_including_receiver)(const char *_Nonnull sourcePath, size_t line);
```

Comment:
Since these have the same signature, you could use the same closure typedef for both functions, maybe named `indexstoredb_file_includes_receiver`

Reply:
done